pkg/version/version.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import "fmt"
+
+var criContainerdVersion = "UNKNOWN"
```

Comment:
Should build version into he binary, see https://github.com/kubernetes/node-problem-detector/pull/71/files#diff-b67911656ef5d18c4ae36cb6741b7965R26.